link_shortener/form_routes.py

```diff
@@ -75,17 +74,14 @@ async def link_password_save(request, link_id):
         link = await check_password(request, link_id, form)
         return redirect(link, status=307)
     except FormInvalidException:
-        status, message = 400, 'Form invalid'
+        status, message = 400, 'invalid-form'
```

Comment:
Tady už pak nikde nepoužíváš ten status. To platí u všech endpointů kde už nepoužíváš template_loader

Reply:
As per slack pokec dane k komentu.